app/assets/javascripts/application.js

```diff
@@ -1,8 +1,6 @@
 //= require jquery.turbolinks
 //= require turbolinks
-//= require jquery.ui.autocomplete
```

Comment:
Apparently katello still uses this in host groups
I've opened an issue - https://projects.theforeman.org/issues/28383